src/integrations/stripe/stripe.ts

```diff
@@ -241,9 +353,17 @@ export function addToCart(
 export function removeFromCart(
   paymentAction: PaymentAction,
   updateFieldValues: any,
-  stripeConfig: any
+  stripeConfig: any,
+  productionEnv: boolean
 ) {
-  const { product_id: productId, clear_cart: clearCart } = paymentAction;
+  const { product_id: configuredProductId, clear_cart: clearCart } =
+    paymentAction;
+  const productId = getLiveOrTestProduct(
+    configuredProductId,
+    stripeConfig,
+    productionEnv
```

Comment:
could we set productionEnv into initState so you can import it directly everywhere instead of needing to pass it? the productionEnv should be consistent for all forms in teh current session anyways